tests/integration/node_unit_tests.rs

```diff
@@ -212,3 +212,7 @@ itest!(unhandled_rejection_web_process {
   envs: env_vars_for_npm_tests(),
   http_server: true,
 });
+
+// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+// The itest macro is deprecated. Please move your new test to ~/tests/specs.
+// !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
Woah, how did it end up here? Did you add it manually?

Reply:
yes I added this because I keep making this mistake